backend/Model/RoomModel.php

```diff
@@ -49,7 +48,10 @@ class RoomModel extends AbstractModel
         'RoomID',
         'Rules',
         'MaxAmount',
-        'DateOfExchange'
+        'DateOfExchange',
+        'UserID',
+        'Rules',
+        'MaxAmount'
```

Comment:
Same as above

Reply:
This one is not duplicated among fillable properties either.
![image](https://user-images.githubusercontent.com/47412010/140475335-18e362e7-55b3-477f-903f-1720626f1165.png)